src/nft/components/explore/Table.tsx

```diff
@@ -98,29 +243,28 @@ export function Table<D extends Record<string, unknown>>({
           prepareRow(row)
 
           return (
-            <TraceEvent
-              events={[Event.onClick]}
-              name={EventName.NFT_TRENDING_ROW_SELECTED}
-              properties={{ collection_address: row.original.collection.address, chain_id: chainId }}
-              element={ElementName.NFT_TRENDING_ROW}
+            <StyledRow
+              {...row.getRowProps()}
               key={i}
+              onClick={() => navigate(`/nfts/collection/${row.original.collection.address}`)}
             >
-              <tr
-                className={styles.tr}
-                {...row.getRowProps()}
-                key={i}
-                onClick={() => navigate(`/nfts/collection/${row.original.collection.address}`)}
-              >
-                {row.cells.map((cell, cellIndex) => {
-                  return (
-                    <td className={clsx(styles.td, classNames?.td)} {...cell.getCellProps()} key={cellIndex}>
-                      {cellIndex === 0 ? <span className={styles.rank}>{i + 1}</span> : null}
-                      {cell.render('Cell')}
-                    </td>
-                  )
-                })}
-              </tr>
-            </TraceEvent>
+              {row.cells.map((cell, cellIndex) => {
+                return (
+                  <td className={clsx(styles.td, classNames?.td)} {...cell.getCellProps()} key={cellIndex}>
+                    {cellIndex === 0 ? (
+                      <RankCellContainer>
+                        <ThemedText.BodySecondary fontSize="14px" lineHeight="20px">
+                          {i + 1}
+                        </ThemedText.BodySecondary>
+                        {cell.render('Cell')}
```

Comment:
having trouble following, what is this doing?

Reply:
this renders the line number for the cell